src/components/SearchResults.js

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from "react";
+import moment from "moment";
+
+const SearchResults = ({ results }) => {
+  function calculateNumberOfNights(a, b) {
+    return moment(a).diff(moment(b), "days");
+  }
+
+  // let trackedResults = [];
+  // results.forEach((element) => {
+  //   {
+  //     trackedResults.push(false);
+  //     console.log(trackedResults);
+  //   }
+  // });
+
+  // const toggleHighlightClass = () => {};
+
+  const TableRow = ({ result }) => {
```

Comment:
This is the correct approach to solving exercise 16, but you shouldn't really define components inside components (I'm actually a bit surprised that it works).

Could you extract this to a separate file?